pkgs/development/libraries/bencodetools/default.nix

```diff
@@ -21,12 +22,14 @@ stdenv.mkDerivation rec {
       --replace 'python_install_option=""' 'python_install_option="--prefix=$out"'
   '';
 
-  enableParallelBuilding = true;
```

Comment:
This is not default-on in the base `stdenv.mkDerivation`. If you need to turn this off for some reason, please make it explicitly `enableParallelBuilding = false` and add a comment with an explanation. As is, this gets rid of parallelism:

```
build flags: -j2 SHELL=/nix/store/m101dg80ngyjdb02g6jwy80sr7kzj26g-bash-5.2p26/bin/bash
```
->
```
build flags: SHELL=/nix/store/izpf49b74i15pcr9708s3xdwyqs4jxwl-bash-5.2p32/bin/bash
```

Reply:
I mistakenly remembered this as the default option.